src/Conversion/KrnlToLLVM/KrnlUnaryMath.cpp

```diff
@@ -123,6 +123,24 @@ struct MathFunctionName<KrnlAtanhOp> {
   }
 };
 
+template <>
+struct MathFunctionName<KrnlIsInfOp> {
+  static std::string functionName(mlir::Type type) {
+
+#if (__APPLE__)
```

Comment:
nit, could you put the #if inside of the `type.isF32()` test? It would read much better, as it is only the string that differs, not the test.

Reply:
Sure let me update it

Reply:
It does not look aesthetically pleasing once I switch the order but I understand the need for consistency....so I made the update.